be/src/exec/vectorized/aggregator.h

```diff
@@ -145,8 +145,12 @@ class Aggregator : public pipeline::ContextWithDependency {
     Aggregator(const TPlanNode& tnode);
 
     virtual ~Aggregator() {
-        if (_state != nullptr) {
-            close(_state);
+        try {
```

Comment:
Could you explain why this `try catch` is needed?

Reply:
Add explanation in the `Problem Summary` section